### ISSUE TITLE
Kicked Dropping Meseta In Gallon's Shop [GC, Plus]

### ISSUE THREAD

**Issue by TheDiscordian** [fuzziqersoftware/newserv#78]
`"EnableItemTracking": true`

Upon joining into Gallon's Shop with 2 players, I went to Gallon's Shop, played the games for a bit, went to the bank, withdrew 1000 meseta, dropped 1000 meseta at bank, kicked.

```
I 21437 2023-02-18 21:35:37 - [Commands] Received from C-3E (Ira) (version=GC command=60 flag=00)                                                                                          
0000 | 60 00 2C 00 5D 0A 00 00 00 00 00 00 D8 1A D5 C2 | ` , ]                                                                                                                             
0010 | 9E B1 82 C2 04 00 00 00 00 00 00 00 00 00 00 00 |                                                                                                                                   
0020 | 13 00 01 00 E8 03 00 00 02 00 00 00             |                                                                                                                                   
I 21437 2023-02-18 21:35:37 - [Lobby/1B] Player 0 split stack to create ground item 00010013 at 0:(-106.552, -65.3469)                                                                     
[PlayerInventory] Meseta: 0                                                                                                                                                                
[PlayerInventory] 16 items                                                                                                                                                                 
[PlayerInventory]   0 (00010000): 020D00 (Kalki LV12 5/1/5/1 82% 43IQ PB:F (light grey))                                                                                                   
[PlayerInventory]   1 (00010001): 030000 (Monomate x7)                                                                                                                                     
[PlayerInventory]   2 (00010002): 030100 (Monofluid x9)                                                                                                                                    
[PlayerInventory]   3 (00010003): 030600 (Antidote x5)                                                                                                                                     
[PlayerInventory]   4 (00010004): 030001 (Dimate x6)                                                                                                                                       
[PlayerInventory]   5 (00010005): 030700 (Telepipe x6)                                                                                                                                     
[PlayerInventory]   6 (00010006): 030900 (Scape Doll)                                                                                                                                      
[PlayerInventory]   7 (00010007): 030300 (Sol Atomizer x4)                                                                                                                                 
[PlayerInventory]   8 (00010008): 010102 (Psy Armor (1 slot))                                                                                                                              
[PlayerInventory]   9 (00010009): 030900 (Scape Doll)                                                                                                                                      
[PlayerInventory]   10 (0001000A): 010202 (Core Shield)                                                                                                                                    
[PlayerInventory]   11 (0001000B): 000700 (Rifle +3 0/5/0/0/0)                                                                                                                             
[PlayerInventory]   12 (0001000C): 010300 (Knight/Power)                                                                                                                                   
[PlayerInventory]   13 (0001000D): 000601 (Autogun 0/5/5/0/0)                                                                                                                              
[PlayerInventory]   14 (0001000E): 030601 (Antiparalysis x2)                                                                                                                               
[PlayerInventory]   15 (00010011): 030400 (Moon Atomizer x7)                                                                                                                               
I 21437 2023-02-18 21:35:37 - [Commands] Sending to C-3C (Peach) (version=GC command=60 flag=00)                                                                                           
0000 | 60 00 2C 00 5D 0A 00 00 00 00 00 00 D8 1A D5 C2 | ` , ]                                                                                                                             
0010 | 9E B1 82 C2 04 00 00 00 00 00 00 00 00 00 00 00 |                                                                                                                                   
0020 | 13 00 01 00 E8 03 00 00 02 00 00 00             |                                                                                                                                   
I 21437 2023-02-18 21:35:37 - [Commands] Received from C-3E (Ira) (version=GC command=60 flag=00)                                                                                          
0000 | 60 00 10 00 29 03 00 00 FF FF FF FF E8 03 00 00 | `   )                                                                                                                             
W 21437 2023-02-18 21:35:37 - [Server] Error processing client command: player does not have enough meseta                                                                                 
I 21437 2023-02-18 21:35:37 - [Server] Client disconnected: C-3E on fd 28
```

**Comment by fuzziqersoftware** [comment]
Sorry it took so long to get to this. This should be fixed by commit c3aca29d.

**Comment by TheDiscordian** [comment]
No worries, I've been trying to find time to play again, I'll report back with the status once I do.
